### PR TITLE
Feature/alamofire weather

### PR DESCRIPTION
**Pull request by MaksimK193**

### CODE REVIEW

File: SwiftUIStudy/Flows/Weather/WeatherViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    var locationManager: CLLocationManager?
+    var coordinate: CLLocationCoordinate2D?
+    
+    init(networkManager: NetworkManagerProtocol = NetworkManager.shared) {
+        self.networkManager = networkManager
+    }
+}
+
+extension WeatherViewModel: CLLocationManagerDelegate {
+    func checkLocationIsEnable() {
```
Comment (IslamRustamov98) [inline SwiftUIStudy/Flows/Weather/WeatherViewModel.swift:24]: checkLocationIsEnable -> checkLocationIsEnable**d**

File: SwiftUIStudy/Flows/Weather/WeatherViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            break
+        }
+    }
+}
+
+extension WeatherViewModel {
+    func getCurrentWeather() {
+        checkLocationIsEnable()
+        let url = "https://api.weather.yandex.ru/v2/forecast?"
+        let token = "26792f4a-06cb-4c0c-aecd-b5ca965b50ab"
```
Comment (IslamRustamov98) [inline SwiftUIStudy/Flows/Weather/WeatherViewModel.swift:60]: What are the best approaches to store sensitive data?
You can set environment variables and access them here - it's one of the best practices. Try to apply it here.

Stuff like urls, tokens and so on are not supposed to be stored plainly in code.

File: SwiftUIStudy/Flows/Weather/WeatherViewModel.swift (hunk 1 above)
Comment (MaksimK193) [reply SwiftUIStudy/Flows/Weather/WeatherViewModel.swift]: Fixed

File: SwiftUIStudy/Flows/Weather/WeatherViewModel.swift (hunk 2 above)
Comment (MaksimK193) [reply SwiftUIStudy/Flows/Weather/WeatherViewModel.swift]: Fixed

**Pull request merged**